pkgs/top-level/all-packages.nix

```diff
@@ -816,7 +816,7 @@ with pkgs;
 
   enum4linux = callPackage ../tools/security/enum4linux { };
 
-  enum4linux-ng = python3Packages.callPackage ../tools/security/enum4linux-ng { };
+  enum4linux-ng = callPackage ../tools/security/enum4linux-ng { };
```

Comment:
Sending this to by-name hierarchy will be nice!

Reply:
Migration to `by-name` (as RFC 140) is already underway.